synapse/handlers/device.py

```diff
@@ -694,10 +739,18 @@ def user_device_resync(self, user_id):
         origin = get_domain_from_id(user_id)
         try:
             result = yield self.federation.query_user_devices(origin, user_id)
-        except (NotRetryingDestination, RequestSendFailed, HttpResponseException):
-            # TODO: Remember that we are now out of sync and try again
-            # later
-            logger.warning("Failed to handle device list update for %s", user_id)
+        except NotRetryingDestination:
```

Comment:
What was the thought for not logging in this case?

Reply:
Because with retries every 30s it would flood the logs of "Failed to handle device list update for..." lines, one for each user being retried whose homeserver is down. Though this still doesn't look like an ideal solution, but then I'm not sure what the ideal solution would look like (see my concern in the initial comment).

Reply:
Makes sense! 👍